AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/DataStoreConfigurationTests.swift

```diff
@@ -0,0 +1,150 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//
+//  DataStoreConfigurationTests.swift
+//  AWSDataStoreCategoryPluginIntegrationTests
+//
+//  Created by Guo, Rui on 6/15/20.
+//  Copyright © 2020 Amazon Web Services. All rights reserved.
+//
+
+import XCTest
+
+import AmplifyPlugins
+import AWSMobileClient
+import AWSPluginsCore
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class DataStoreConfigurationTests: XCTestCase {
+
+    func testConfigWithSameSchema() throws {
+
+        let databaseCreationDone = expectation(description: "Old database deleted and new database recreated")
+
+        do {
+            let dataStorePlugin = AWSDataStorePlugin(modelRegistration: AmplifyModels())
```

Comment:
nit: be explicit then `let previousVersion = 123` and use previousVersion across both configurations in this test method

Reply:
changed, plz take a look